layouts/_partials/navbar.html

```diff
@@ -20,16 +14,7 @@
   ></div>
 
   <nav class="hextra-max-navbar-width hx:mx-auto hx:flex hx:items-center hx:justify-end hx:gap-2 hx:h-16 hx:px-6">
-    <a class="hx:flex hx:items-center hx:hover:opacity-75 hx:ltr:mr-auto hx:rtl:ml-auto" href="{{ $logoLink }}">
-      {{- $displayTitle := (.Site.Params.navbar.displayTitle | default true) }}
-      {{- if (.Site.Params.navbar.displayLogo | default true) }}
-        <img class="hx:mr-2 hx:block hx:dark:hidden" src="{{ $logoPath | relURL }}" alt="{{ cond $displayTitle `Logo` .Site.Title }}" height="{{ $logoHeight }}" width="{{ $logoWidth }}" />
-        <img class="hx:mr-2 hx:hidden hx:dark:block" src="{{ $logoDarkPath | relURL }}" alt="{{ cond $displayTitle `Dark Logo` .Site.Title }}" height="{{ $logoHeight }}" width="{{ $logoWidth }}" />
-      {{- end }}
-      {{- if $displayTitle }}
-        <span class="hx:mr-2 hx:font-extrabold hx:inline hx:select-none" title="{{ .Site.Title }}">{{- .Site.Title -}}</span>
-      {{- end }}
-    </a>
+    {{ partial "title.html" . }}
```

Comment:
I think `title.html` might be a bit too generic and could lead to confusion. I’d suggest using `navbar-title.html` instead.

Reply:
Fixed.